<?xml version="1.0"?>
<launch>
    <arg name="robot_ns"  default="$(env ROBOT_NS)"/>
    <arg name="tf_prefix"  default="$(env ROBOT_TF_PREFIX)"/>
    <group ns="$(arg robot_ns)">
        <node pkg="docking" type="docking.py" name="docking" output="screen"/>
        <node pkg="docking" type="Navigate_back_to_base_real_stop_and_listen.py" name="nav_to_base" output="screen">
            <rosparam file="$(find docking)/config/nav_to_base.yaml" command="load"/>
        </node>
        <node pkg="docking" type="control.py" name="control" output="screen"/>
    </group>
</launch>